api/user.go

```diff
@@ -4,24 +4,36 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"os"
 
+	"github.com/netsage-project/grafana-dashboard-manager/config"
 	"github.com/netsage-project/sdk"
 	"github.com/sirupsen/logrus"
 )
 
+func validateUserAPI(client *sdk.Client) {
+	if client == nil || !config.GetDefaultGrafanaConfig().AdminEnabled {
+		logrus.Info("Missing Admin client, please check your config and ensure basic auth is configured")
```

Comment:
logrus has a Fatal method: https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

Reply:
I was trying to avoid that monster stack trace.  Does fatal just terminate after the message?